### ISSUE TITLE
Missing alt text for post author avatar

### ISSUE THREAD

**Issue by t-hamano** [Automattic/o2#240]
Originally reported at: https://github.com/WordPress/gutenberg/issues/70726

<img width="1307" height="640" alt="Image" src="https://github.com/user-attachments/assets/523e3a32-c332-420d-b028-cf3156d318ae" />

As far as I have checked, there is no alt value in the following two places:

https://github.com/Automattic/o2/blob/271d520d4f118f382164f1492fd9fd11f8529153/inc/tpl/post-edit.php#L5

https://github.com/Automattic/o2/blob/271d520d4f118f382164f1492fd9fd11f8529153/inc/tpl/post.php#L5

**Comment by sabernhardt** [comment]
Also in the x-post template, the avatar is linked without any alternative text (or `alt` attribute):
https://github.com/Automattic/o2/blob/271d520d4f118f382164f1492fd9fd11f8529153/inc/tpl/xpost.php#L2-L4
